src/xdocs/checks/naming/abstractclassname.xml.template

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I went with `xxx.xml.template`, because doxia looks for everything that is `*.xml`. I tried with `xxx.template.xml` and it generated an `.html` file when `mvn site` is ran.

We can also do `xxx.template`. 

Let me know what you prefer. Either is fine with me.

Reply:
I am good with `xxx.xml.template`

Reply:
`xxx.xml.template` is good

Reply:
`xxx.xml.template` is good